service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -217,6 +223,42 @@ void shouldNotifyPartiesOnC21OrderSubmission() throws IOException, NotificationC
         }
     }
 
+    @Nested
+    class CaseManagementOrderNotificationTests {
+        final String subjectLine = "Lastname, SACCCCCCCC5676576567";
+        final Map<String, Object> cmoOrderIssuedNotificationParameters = getCMOIssuedNotificationParameters();
+
+        @BeforeEach
+        void setup() throws IOException {
+            given(inboxLookupService.getNotificationRecipientEmail(callbackRequest().getCaseDetails(),
+                LOCAL_AUTHORITY_CODE))
+                .willReturn(LOCAL_AUTHORITY_EMAIL_ADDRESS);
+
+            given(caseManagementOrderEmailContentProvider.buildCMOIssuedNotificationParametersForLocalAuthority(
+                callbackRequest().getCaseDetails(), LOCAL_AUTHORITY_CODE))
+                .willReturn(cmoOrderIssuedNotificationParameters);
+        }
+
+        @Test
+        void shouldNotifyLocalAuthorityOfCaseManagementOrderIssued() throws Exception {
```

Comment:
I think it's fine to abbreviate CaseManagementOrder here. `shouldNotifyLocalAuthorityOfIssuedCMO`